### PR TITLE
Handle data submission in offline state

### LINKED ISSUE #1133: Handle data submission in offline state
STR:
- go offline (added by @thisandagain)
- choose survey app
- tap start building
- scroll to the bottom
- tap the `Submit` button

Nothing happens (and no data is visible in the app's Data tab)

### PR DESCRIPTION
**Pull request by gesa**
Fixes #1133

### CODE REVIEW

**Comment by alicoding** [comment]
I'm adding `blocked` label for l10n in this file since I can't upload and asking the contributors to translate again frequently.

**Comment by thisandagain** [comment]
:+1:

File: components/publishingStatus/index.js [hunk 1]
```diff
@@ -28,5 +31,13 @@ module.exports = {
                 self.$data.isPublishing = isPublishing = false;
             }, 5000);
         });
+        this.$on('publishOffline', function () {
+            self.$data.isOnline = false;
+            self.$data.isPublishing = true;
+            console.log('bar');
```
Comment (alicoding) [inline components/publishingStatus/index.js]: console.log :P

**Comment by thisandagain** [comment]
I'd prefer to land this (in english) and hold off on updating the Transifex file till after MWC. Thoughts @alicoding ?

**Comment by alicoding** [comment]
@thisandagain if this patch is not going to be merge and ready in an hour then I would say we hold off on updating this on Transifex, but either way I have to update the file on Transifex anyway since we found few unlocalized strings and they have to be localized.

File: components/publishingStatus/index.js (hunk 1 above)
Comment (gesa) [inline components/publishingStatus/index.js]: Good catch ^_^

**Comment by gesa** [comment]
I fixed that `console.log()`, should be good to go.

File: components/publishingStatus/index.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ var analytics = require('../../lib/analytics');
 module.exports = {
     className: 'publishing-status',
     template: require('./index.html'),
+    data: {
+        isOnline: undefined
```
Comment (alicoding) [inline components/publishingStatus/index.js]: set this to `true` instead of `undefined`.

**Comment by alicoding** [comment]
r+

**Pull request merged**